tests/unit_tests/plugins/modules/output_checkers/test_output_checkers.py

```diff
@@ -0,0 +1,40 @@
+# # # Distribution Statement A. Approved for public release. Distribution unlimited.
+# # #
+# # # Author:
+# # # Naval Research Laboratory, Marine Meteorology Division
+# # #
+# # # This program is free software: you can redistribute it and/or modify it under
+# # # the terms of the NRLMMD License included with this program. This program is
+# # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+# # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+# # # for more details. If you did not receive the license, for more information see:
+# # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+"""Test all Output Checker plugins."""
+import pytest
+
+from geoips.interfaces import output_checkers
+from geoips.commandline import log_setup
+
+log_setup.setup_logging()
+
+
+class TestOutputCheckers:
+    """TestOutputChecker class, defining methods as well."""
+
+    available_output_checkers = ["geotiff", "image", "netcdf", "text"]
```

Comment:
@evrose54 @AJThorpe @jsolbrig Do we want to use entry points here before we merge?  So we are not explicitly listing all of the plugins in the unit test infrastructure?  That way it will automatically pick up plugins in other repos, as long as they have the right attributes.

Reply:
I'll leave this up to everyone else. I was thinking no, since some output checkers might want different unit tests, but that may not be the case.

Reply:
I think we can still implement different unit tests, even if we automatically identify all the output checker plugins here.  If the unit test attributes aren't defined, this will xfail (which I think is fine), and a separate unit test can be implemented.

We *definitely* want all the installed output checkers to be automatically discovered, so that they all can utilitze the base infrastructure without having to explicitly add the plugin name here.

Reply:
I just changed this to be discovered via entry points.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/379/files#r1373579423
Fixed there.